### ISSUE TITLE
Note fails to import, but unable to narrow down cause

### ISSUE THREAD

**Issue by upnix** [akosbalasko/yarle#416]
This note will fail to import:
[TestNote.enex.zip](https://github.com/akosbalasko/yarle/files/9944371/TestNote.enex.zip)

With the message in `conversion.log`:
```
Conversion started at Sat Nov 05 2022 11:27:34 GMT-0600 (Mountain Daylight Time)
Converting note "Aug. 4 - 9"...
Failed to convert note: Aug. 4 - 9, {}
Conversion finished at Sat Nov 05 2022 11:27:34 GMT-0600 (Mountain Daylight Time)
Note "Aug. 4 - 9" converted successfully in 0.136 seconds.
Notes processed: 1
```

In Evernote, if I make any change to the note, then re-export, Yarle is able to import the note. Examining the re-exported note I notice that there are _a lot_ more `<div>`'s around content than the original export. But I can't determine why Yarle is unhappy with the fewer `<div>` export.

Here is the same note with only an extra new line added in Evernote before export: 
[TestNote_2.enex.zip](https://github.com/akosbalasko/yarle/files/9944377/TestNote_2.enex.zip)

Here is the entire contents of `conversion.log`, but I changed no settings from the defaut:
[conversion.log.txt](https://github.com/akosbalasko/yarle/files/9944386/conversion.log.txt)

Sorry I wasn't able to narrow it down more.

**Comment by akosbalasko** [comment]
Hi @upnix ,
Thanks for the report and for the example enex file, I'll investigate the situation soon.
akos

**Comment by upnix** [comment]
I keep coming across this bug. I think the difference between the note that imports and the note that doesn't comes down to the `<en-todo/>` tag.  Every note that fails to import will have checkboxes.

In the note that fails to import, unchecked boxes precede text with `<en-todo/>`.  

The note that _will_ import replaces these with `<en-todo checked="false" />`

**Comment by akosbalasko** [comment]
Hi @upnix ,
Thank you for the investigation, I fixed the issue and released a new version(4.5.1). 
I close this issue but do not hesitate to open a new one if you still see problems. 
Thanks a lot!
